scripts/zones/Chocobo_Circuit/Zone.lua

```diff
@@ -12,6 +12,9 @@ end
 
 zone_object.onZoneIn = function(player, prevZone)
     local cs = -1
+	if (player:getXPos() == 0 and player:getYPos() == 0 and player:getZPos() == 0) then
```

Comment:
Please align this properly. 4 spaces, no tabs.
Also, please remove parenthesis ( ) containing the "if". Those are unneeded